Source/Core/VideoCommon/VertexLoaderX64.cpp

```diff
@@ -53,7 +53,7 @@ VertexLoaderX64::VertexLoaderX64(const TVtxDesc& vtx_desc, const VAT& vtx_att)
   GenerateVertexLoader();
   WriteProtect();
 
-  const std::string name = ToString();
+  std::string name = fmt::format("VertexLoaderX64\nVtx desc: \n{}\nVAT:\n{}", vtx_desc, vtx_att);
```

Comment:
Why was the `const` removed here?

Reply:
I don't think there's any reason why; it probably was by accident but I'm not entirely sure how I managed to do that.